modules/languages/php.nix

```diff
@@ -0,0 +1,75 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib;
+with builtins; let
+  cfg = config.vim.languages.php;
+in {
+  options.vim.languages.php = {
+    enable = mkEnableOption "PHP language support";
+
+    treesitter = {
+      enable = mkEnableOption "Enable PHP treesitter" // {default = config.vim.languages.enableTreesitter;};
+      package = nvim.types.mkGrammarOption pkgs "php";
+    };
+
+    lsp = {
+      enable = mkEnableOption "PHP LSP support" // {default = config.vim.languages.enableLSP;};
+
+      package = mkOption {
+        description = "lsp pacakge";
+        type = types.package;
+        default = pkgs.phpactor;
+      };
+    };
+
+    analyzer = {
+      package = mkOption {
+        description = "analizer pacakge";
+        type = types.package;
+        default = pkgs.php81Packages.phan;
+      };
+    };
+  };
+
+  config = mkIf cfg.enable (mkMerge [
+    (mkIf cfg.treesitter.enable {
+      vim.treesitter.enable = true;
+      vim.treesitter.grammars = [cfg.treesitter.package];
+    })
+    (mkIf cfg.lsp.enable {
+      vim = {
+        lsp.lspconfig.enable = true;
+        lsp.lspconfig.sources.php-lsp = ''
+          require'lspconfig' = {
+            phpactor.setup{
+              cmd = {
+                "${cfg.lsp.package}/bin/phpactor",
+                "language-server"
+              },
```

Comment:
with the recent versions, we're trying to move off hardcoded lsp commands - please take a look at other language modules to see the alternative

t